tests/test_events.py

```diff
@@ -415,3 +415,18 @@ def test_delete_event(event, client, admin_client):
 
     event_in_db = LAMetroEvent.objects.filter(id=e.id)
     assert not event_in_db.exists()
+
+
+# test event (event location name = TEST; get 404)
+def test_private_event(client, event, mocker):
+    private_event = event.build()
+
+    mock_location = mocker.MagicMock(spec=EventLocation)
+    mock_location.name = 'TEST'
+
+    mocker.patch('lametro.models.LAMetroEvent.location', return_value=mock_location)
```

Comment:
I tried mocking the location here, and it seems to be working, but in remembering the rule to patch where the property is called, there didn't seem to be a good place in the code to patch this single event so that it has the mocked properties. The best place I found to patch would be in the EventManager's `get_queryset` method, but patching over a whole queryset doesn't seem right. Need to rethink my approach here.
https://github.com/datamade/la-metro-councilmatic/blob/24cb5635b6958ade2bd9ded2c041dc2b64c1e4ee/lametro/models.py#L393-L396

Reply:
I'm going to try to create an `EventLocation` testing fixture that I could use to then give `private_event` a location property.